### PR TITLE
Liams adventure

### PR DESCRIPTION
**Pull request by wkwyatt**
Added a story link where you bite the marshmallow wall and some interesting things happen, also links back to original story.

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2004)

### CODE REVIEW

**Comment by wkwyatt** [comment]
Please review this, @casey-collab

File: bite-the-marshmallow/tasty-marshmallow.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+"Mmmmm... these marshmallows are soooo good" 
+
+You close your eyes.
+
+Wait a minute....how did you end up in this car?  You must be hallucinating 
+
+[You use the gps to get back home](https://www.youtube.com/watch?v=RvlhrgSZrfA)
+
+[You open the door and there is a man sitting on your couch](../movie-ripoffs/zoolander/zoolander.md)
```
Comment (casey-collab) [inline bite-the-marshmallow/tasty-marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: bite-the-marshmallow/tasty-marshmallow.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+"Mmmmm... these marshmallows are soooo good"
```
Comment (casey-collab) [inline bite-the-marshmallow/tasty-marshmallow.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**